frontend/app/(groups)/g/dashboard/group-selector.tsx

```
@@ -6,6 +6,7 @@ import {
   PlusCircle,
   PlusCircleIcon,
   ShieldIcon,
+  StarIcon,
```

ESLint

[error] 'StarIcon' is defined but never used.

(@typescript-eslint/no-unused-vars)